app/src/main/java/ch/hikemate/app/utils/DatetimeUtils.kt

```diff
@@ -47,7 +53,7 @@ fun Timestamp.humanReadablePlannedLabel(
     locale: Locale = Locale.getDefault(),
     currentDate: LocalDate = LocalDate.now()
 ): String {
-  val plannedDate = this.toLocalDate()
+  val plannedDate = this.toLocalDate(locale = locale)
```

Comment:
Suggestion: we should probably not pass anything here, so that the default system value is used. Mapping a locale to a zone id is hard to get right, we can leave it to the system.

```suggestion
  val plannedDate = this.toLocalDate()
```